code/modules/clothing/under/miscellaneous.dm

```diff
@@ -385,67 +277,6 @@
 	item_state = "kilt"
 	worn_state = "plaid_purple"
 
-//wedding stuff
-/obj/item/clothing/under/wedding/
```

Comment:
i dont suppose crew members might ever get married? Seems like a fun IC event for necromorphs to interrupt
even if unlikely, i'd rather keep at least one of these around

Reply:
That seems very LRP for the grimdark attitude of the game.